src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -60,33 +60,31 @@ const TransferDescription = ({ value = '', symbol, recipient }: TransferDescProp
       {' '}
       to:
     </Bold>
-    <br />
     <EtherscanLink type="address" value={recipient} />
-  </Paragraph>
+  </Block>
 )
 
 const SettingsDescription = ({ removedOwner, addedOwner, newThreshold }: DescriptionDescProps) => (
   <>
     {removedOwner && (
-      <Paragraph data-testid={TRANSACTIONS_DESC_REMOVE_OWNER_TEST_ID}>
+      <Block data-testid={TRANSACTIONS_DESC_REMOVE_OWNER_TEST_ID}>
         <Bold>Remove owner:</Bold>
-        <br />
         <EtherscanLink type="address" value={removedOwner} />
-      </Paragraph>
+      </Block>
     )}
     {addedOwner && (
-      <Paragraph data-testid={TRANSACTIONS_DESC_ADD_OWNER_TEST_ID}>
+      <Block data-testid={TRANSACTIONS_DESC_ADD_OWNER_TEST_ID}>
         <Bold>Add owner:</Bold>
-        <br />
         <EtherscanLink type="address" value={addedOwner} />
-      </Paragraph>
+      </Block>
     )}
     {newThreshold && (
-      <Paragraph data-testid={TRANSACTIONS_DESC_CHANGE_THRESHOLD_TEST_ID}>
+      <Block data-testid={TRANSACTIONS_DESC_CHANGE_THRESHOLD_TEST_ID}>
```

Comment:
Why block? This is text

Reply:
It has the component `EtherscanLink`, which has the component `Block` as his root element